src/js/actions/layers.js

```diff
@@ -1451,14 +1452,24 @@ define(function (require, exports) {
         
         return descriptor.playObject(createObj)
             .bind(this)
-            .then(function () {
-                log.debug("Warning: calling updateDocument to add a single artboard is very slow!");
-                return this.transfer(documentActions.updateDocument);
+            .then(function (result) {
+                var payload = {
+                    documentID: document.id,
```

Comment:
withdrawn.  it looks/looked weird to my eye, but a quick grep proved that this is an established pattern (for simple objects too, not just functions)